client/src/app/components/filters/filters.component.html

```diff
@@ -0,0 +1,12 @@
+<ion-segment value="all">
+  <ion-segment-button *ngFor="let topic of topics" value="{{ topic }}">
```

Comment:
```suggestion
  <ion-segment-button *ngFor="let topic of topics" [value]="topic">
```

When providing a variable to an HTML-attribute, you need to use the square-brackets. (so it is clear to see that something 'dynamic' is used)

See: https://angular.io/guide/property-binding